app/src/main/java/com/example/ubuntu/digitalwatchface/model/WatchFaceStyleModels.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.ubuntu.digitalwatchface.model
+
+//no background image so resource id = 0
+const val EMPTY_IMAGE_RESOURCE = 0
+
+data class DigitalWatchFaceStyle(
+        val watchFaceColors: WatchFaceColors,
+        val watchFaceDimensions: WatchFaceDimensions,
+        val watchFaceBackgroundImage: WatchFaceBackgroundImage
+
+)
+
+
+data class WatchFaceColors(
+        //main color for clock text
+        val main:Int,
```

Comment:
I don't think the comment above the variable name is necessary, the variable name should be descriptive enough..
For instance this variable name can be called `val mainColorForClockText:Int`

Reply:
ok I'll change it to mainTextColor, but before that should I finish my cherry pick first? If I change it now it might disrupt my code while adding the other cherry pick commits

Reply:
oh yeah you can, I thought this was a request to a temporary develop branch though